cirq-aqt/cirq_aqt/aqt_device.py

```diff
@@ -305,9 +293,9 @@ def at(self, position: int) -> Optional[cirq.LineQubit]:
 
     def _value_equality_values_(self) -> Any:
         return (
-            self._measurement_duration,
-            self._twoq_gates_duration,
-            self._oneq_gates_duration,
+            self.metadata.measurement_duration,
+            self.metadata.twoq_gates_duration,
+            self.metadata.oneq_gates_duration,
```

Comment:
Optional:  Do you think it is worth just replacing this with self.metadata?

Reply:
Sure, that is much cleaner, thanks!  Done in fa56cc7874769c844c358151eb1d9771339ed4fc.